src/VisualStudio/Core/Def/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -282,15 +261,26 @@ public void ReportError2(
         int iEndColumn,
         string bstrFileName)
     {
```

Comment:
Just to call it out, if the concern was also the exceptions being thrown, there's a comment on this method about switching it to be PreserveSig. Not sure that changes any of the deeper discussons here. That might involve changes to our COM types though that might break the CPS code if that feature flag wasn't enabled.